padle/padle.py

```diff
@@ -0,0 +1,537 @@
+import discord
+import datetime
+import random
+import asyncio
+import logging
+
+from typing import List, Literal, Optional
+from redbot.core import Config, commands
+from math import ceil
+
+# from dbcog.monster_graph import MonsterGraph # used for creating the filter
```

Comment:
you're going to have to do a cross-cog typing import for this. Import `TYPE_CHECKING` from `typing`, and do something along the lines of 

```py
if TYPE_CHECKING:
    <a bunch of import statements from other cogs>
```